src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -436,49 +438,51 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
-      createEsqlRuleType: `${SECURITY_SOLUTION_DOCS}rules-ui-create.html#create-esql-rule`,
-      ruleUiAdvancedParams: `${SECURITY_SOLUTION_DOCS}rules-ui-create.html#rule-ui-advanced-params`,
+      privileges: `${ELASTIC_DOCS}solutions/security/configure-elastic-defend/elastic-defend-feature-privileges`,
+      manageDetectionRules: `${ELASTIC_DOCS}solutions/security/detect-and-alert/manage-detection-rules`,
+      createDetectionRules: `${ELASTIC_DOCS}solutions/security/detect-and-alert/create-detection-rule`,
+      updatePrebuiltDetectionRules: `${ELASTIC_DOCS}solutions/security/detect-and-alert/install-manage-elastic-prebuilt-rules#update-prebuilt-rules`,
+      prebuiltRuleCustomizationPromoBlog: isServerless
+        ? '' // URL for Serverless to be added later, once the blog post is published. Issue: https://github.com/elastic/kibana/issues/209000
+        : `https://www.elastic.co/blog/security-prebuilt-rules-editing`,
+      createEsqlRuleType: `${ELASTIC_DOCS}solutions/security/detect-and-alert/create-detection-rule#create-esql-rule`,
+      ruleUiAdvancedParams: `${ELASTIC_DOCS}solutions/security/detect-and-alert/create-detection-rule#rule-ui-advanced-params`,
       entityAnalytics: {
-        riskScorePrerequisites: `${SECURITY_SOLUTION_DOCS}ers-requirements.html`,
-        entityRiskScoring: `${SECURITY_SOLUTION_DOCS}entity-risk-scoring.html`,
-        assetCriticality: `${SECURITY_SOLUTION_DOCS}asset-criticality.html`,
+        riskScorePrerequisites: `${ELASTIC_DOCS}solutions/security/advanced-entity-analytics/entity-risk-scoring-requirements`,
+        entityRiskScoring: `${ELASTIC_DOCS}solutions/security/advanced-entity-analytics/entity-risk-scoring`,
+        assetCriticality: `${ELASTIC_DOCS}solutions/security/advanced-entity-analytics/asset-criticality`,
       },
-      detectionEngineOverview: `${SECURITY_SOLUTION_DOCS}detection-engine-overview.html`,
-      aiAssistant: `${SECURITY_SOLUTION_DOCS}security-assistant.html`,
-      signalsMigrationApi: `${SECURITY_SOLUTION_DOCS}signals-migration-api.html`,
-      legacyEndpointManagementApiDeprecations: `${KIBANA_DOCS}breaking-changes-summary.html#breaking-199598`,
-      legacyRuleManagementBulkApiDeprecations: `${KIBANA_DOCS}breaking-changes-summary.html#breaking-207091`,
-      siemMigrations: `${SECURITY_SOLUTION_DOCS}siem-migration.html`,
-      llmPerformanceMatrix: `${SECURITY_SOLUTION_DOCS}llm-performance-matrix.html`,
+      detectionEngineOverview: `${ELASTIC_DOCS}solutions/security/detect-and-alert`,
+      aiAssistant: `${ELASTIC_DOCS}solutions/security/ai/ai-assistant`,
+      signalsMigrationApi: isServerless
+        ? `${KIBANA_APIS}group/endpoint-security-detections-api`
+        : `${KIBANA_SERVERLESS_APIS}group/endpoint-security-detections-api`,
+      legacyEndpointManagementApiDeprecations: `${ELASTIC_DOCS}release-notes/security/breaking-changes`,
+      legacyRuleManagementBulkApiDeprecations: `${ELASTIC_DOCS}release-notes/security/breaking-changes`,
```

Comment:
>      legacyRuleManagementBulkApiDeprecations: `${ELASTIC_DOCS}release-notes/security/breaking-changes`,

#207091 is not mentioned explicitly in https://www.elastic.co/docs/release-notes/security/breaking-changes and there are no anchors so I've just linked to the page.

Reply:
Removed in 11cb4f2f51443d570b4ad828ba0709a8f0222026 since it's no longer used anywhere